lib/reckon/app.rb

```diff
@@ -115,7 +119,8 @@ def output(ledger_line)
       options[:output_file].flush
     end
 
-    def guess_account(row)
+    # Weigh accounts by how well they match the row
+    def weighted_account_match( row )
```

Comment:
Looks like a didn't write a spec about this method.  Oops :)  Up to you if you want to backfill one, it's okay with me if you don't.